### PR TITLE
Infra/test kit add props

### PR DESCRIPTION
**Pull request by M-i-k-e-l**
## Description
Add props to the testkit - this improves the quality of the tests (already found some issues).
I'm not sure about the `TextField`'s `isPressable` being removed but we can add it later (it's wrong anyway since `onPress` does not exist).
Relates to WOAUILIB-2351

## Changelog
None

## Additional info
None

### CODE REVIEW

File: src/testkit/index.ts [hunk 1]
```diff
@@ -11,3 +11,4 @@ export {HintDriver} from '../components/hint/Hint.driver';
 export {RadioButtonDriver} from '../components/radioButton/RadioButton.driver';
 export {RadioGroupDriver} from '../components/radioGroup/RadioGroup.driver';
 export {SortableListItemDriver} from '../components/sortableList/SortableListItem.driver';
+export {SliderDriver} from '../components/slider/slider.driver';
```
Comment (Inbal-Tish) [inline src/testkit/index.ts:14]: better use the incubator slider driver. The old Slider will be replaced on the next major version

File: src/components/slider/slider.driver.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
+import {SliderProps} from './index';
 import {ComponentDriver} from '../../testkit';
 
-export class SliderDriver extends ComponentDriver {
-  isDisabled = async () => (await this.getElementProps()).accessibilityStates[0] === 'disabled';
+export class SliderDriver extends ComponentDriver<SliderProps> {
+  isDisabled = async () => (await this.getElementProps()).accessibilityState?.disabled === true;
```
Comment (Inbal-Tish) [inline src/components/slider/slider.driver.ts:5]: Did you check if it works? I remember the array did work...

File: src/components/slider/index.tsx [hunk 3]
```diff
@@ -259,7 +259,7 @@ export default class Slider extends PureComponent<SliderProps, State> {
       accessibilityLabel: 'Slider',
       accessible: true,
       accessibilityRole: 'adjustable' as AccessibilityRole,
-      accessibilityStates: disabled ? ['disabled'] : [],
+      accessibilityState: disabled ? {disabled} : undefined,
```
Comment (Inbal-Tish) [inline src/components/slider/index.tsx:262]: Did you check if it works? I remember the array did work...

File: src/testkit/index.ts (hunk 1 above)
Comment (M-i-k-e-l) [reply src/testkit/index.ts:14]: There is no driver for the incubator slider, or did I miss it?
I might write it in the next phase and replace this

File: src/components/slider/index.tsx (hunk 3 above)
Comment (M-i-k-e-l) [reply src/components/slider/index.tsx:262]: Yes

File: src/testkit/index.ts (hunk 1 above)
Comment (Inbal-Tish) [reply src/testkit/index.ts:14]: Right, my bad

**Review by Inbal-Tish — APPROVED** [review]

**Pull request merged**